src/main/scala/com/sumologic/sumobot/plugins/BotPlugin.scala

```diff
@@ -77,16 +78,19 @@ abstract class BotPlugin
   protected def sendImage(im: OutgoingImage): Unit = context.system.eventStream.publish(im)
 
   class RichIncomingMessage(msg: IncomingMessage) {
-    def response(text: String) = OutgoingMessage(msg.channel, responsePrefix + text)
+    def response(text: String, inThread: Boolean = false) = {
+      val threadTs = if (inThread) { Some(msg.idTimestamp) } else { None }
```

Comment:
`Ts` was not an obvious abbreviation to me.  I think spelling it out (`Timestamp`) would be better

Reply:
`thread_ts` is what they call it in Slack API, e.g. https://api.slack.com/events/message/message_replied

They don't even say it loud that `ts` stands for `timestamp`.

Reply:
Oh.  huh.  Ok, then yeah, best to be consistent with the official API.